src/gwt/src/org/rstudio/core/client/layout/LogicalWindow.java

```diff
@@ -125,6 +125,13 @@ public WindowState getState()
       return state_;
    }
 
+   public boolean equals(LogicalWindow window)
+   {
+      if(normal_.equals(window.normal_))
```

Comment:
Maybe simplify to `return normal_.equals(window.normal_)`?

Reply:
Good call, fixed